project/Dependencies.scala

```diff
@@ -6,16 +6,17 @@ object Dependencies {
   val play25Version = "2.5.10"
   val play26Version = "2.6.7"
 
-  def playJson(playVersion: String, scalaVersion: String): ModuleID = {
-    "com.typesafe.play" % s"play-json_$scalaVersion" % playVersion
+  def playJson(playVersion: String): ModuleID = {
+    "com.typesafe.play" %% s"play-json" % playVersion
```

Comment:
I did this with `%%`, the scala version will be injected (we need it when cross compiling)

Reply:
This no longer compiles: 
```
sbt.librarymanagement.ResolveException: unresolved dependency: com.typesafe.play#play-ws_2.12;2.3.10: not found

```